remake.yml

```diff
@@ -34,6 +34,8 @@ targets:
   state_tasks:
     command: do_state_tasks(oldest_active_sites, '1_fetch/src/get_site_data.R')
     depends: parameter
+    depends: '3_visualize/src/plot_site_data.R'
+    depends: '2_process/src/tally_site_obs.R'
```

Comment:
Hmm, I worry that **scipiper** might not treat all of these `depends` elements the same. I haven't actually experimented with doing it the way you have, but when you need multiple dependencies, the way I know does work is as you've done it for the default target, more like this:
```yml
  main:
    depends:
      - 3_visualize/out/site_map.png
      - 3_visualize/out/data_coverage.png
      - 3_visualize/out/timeseries_map.html
```
(one `depends:` with multiple items with dashes that follow).

However, also see my earlier comment about that last check box in the last open issue comment - once you've addressed that subtask, this code block should look different anyway.